README.md

```diff
@@ -496,6 +496,18 @@ HostbasedAuthentication: Enables/disables Hostbased authentication. Valid values
 
 - *Default*: 'no'
 
+sshd_ignoreuserknownhosts
+-------------------------
+IgnoreUserKnownHosts: Enables/disables Hostbased authentication. Valid values are 'yes' and 'no'.
+
+- *Default*: 'no'
+
+sshd_ignorerhosts
+-------------------------
+IgnoreRhosts: Enables/disables Hostbased authentication. Valid values are 'yes' and 'no'.
```

Comment:
String for IgnoreRhosts option in sshd_config. Valid values are 'yes' and 'no. Specifies that `.rhosts` and `.shosts` files will not be used in RhostsRSAAuthentication or HostbasedAuthentication though `/etc/hosts.equiv` and `/etc/ssh/shosts.equiv` are still used.